lib/rules/no-extra-parens.js

```diff
@@ -67,6 +68,9 @@ module.exports = {
         const NESTED_BINARY = ALL_NODES && context.options[1] && context.options[1].nestedBinaryExpressions === false;
         const EXCEPT_RETURN_ASSIGN = ALL_NODES && context.options[1] && context.options[1].returnAssign === false;
         const IGNORE_JSX = ALL_NODES && context.options[1] && context.options[1].ignoreJSX;
+        const ENFORCE_FOR_ARROW_CONDITIONALS = ALL_NODES && context.options[1] &&
```

Comment:
The option now behaves correctly, but the name of this variable is misleading because it does the opposite of what the option does. One solution would be to rename the *variable* to `IGNORE_ARROW_CONDITIONALS`.